girder/utility/plugin_utilities.py

```diff
@@ -35,11 +35,14 @@
 import sys
 import traceback
 import yaml
+import importlib
+
+from pkg_resources import iter_entry_points
 
 from girder.constants import PACKAGE_DIR, ROOT_DIR, ROOT_PLUGINS_PACKAGE, \
     TerminalColor
 from girder.models.model_base import ValidationException
-from girder.utility import config, mail_utils, mkdir
+from girder.utility import config as _config, mail_utils, mkdir
```

Comment:
Hm, this makes me want to rename our new symbol to `pluginConfig` to be less vague...